source/common/upstream/cluster_manager_impl.h

```diff
@@ -36,7 +36,8 @@ class ProdClusterManagerFactory : public ClusterManagerFactory {
                             Ssl::ContextManager& ssl_context_manager,
                             Event::Dispatcher& primary_dispatcher,
                             const LocalInfo::LocalInfo& local_info)
-      : runtime_(runtime), stats_(stats), tls_(tls), random_(random), dns_resolver_(dns_resolver),
+      : runtime_(runtime), stats_(stats), cds_scope_(stats.createScope("cluster_manager.cds.")),
```

Comment:
We can put the `ScopePtr` creation and ownership inside the `CdsApiImpl`, but then we have multiple scopes for each CDS instance. I guess it's technically cleaner, since you only pay the price when doing CDS.